scripts/golangci.yml.in

```diff
@@ -169,3 +169,4 @@ linters:
     - testpackage
     # TODO: enable goerr113, see: https://github.com/ceph/ceph-csi/issues/1227
     - goerr113
+    - nolintlint
```

Comment:
I don't think we can accept this. Our e2e code should have the same quality as the rest of the project.

Reply:
> When a worker node does not have any VolumeAttachments with type `mounter: rbd-nbd`, the starting of the NodeServer should not get much delayed.

Completely agree with Niels here. I don't think we should have an option to enable Volume Healer.

Reply:
Sure, I had fixed it the other way now.